hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterAdminCLI.java

```diff
@@ -1918,6 +1920,9 @@ public void testAddMultipleMountPointsFailure() throws Exception {
             "-faulttolerant"};
     // mount points were already added
     assertNotEquals(0, ToolRunner.run(admin, argv));
+
+    assertTrue("The error message should return failed entries",
+        err.toString().contains("Cannot add mount points: [0SLASH0testAddMultiMountPoints-01"));
```

Comment:
`/` and `:` are replaced here: https://github.com/apache/hadoop/blob/trunk/hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/store/driver/impl/StateStoreSerializableImpl.java#L104

Can you create a small function to reverse that so the keys in the error message are readable

Reply:
Sounds good, that will be more readable to operators. Let me make this change.